test-typings/delay.ts

```diff
@@ -100,3 +100,46 @@ import { delay } from '../src/delay';
 
   expectType<Event<number>>(delay({ source, timeout: 100, target }));
 }
+
+// supports multiple targets as an array
+{
```

Comment:
Please, add tests for different types. For example, it really shows an error if use `source: Event<number>` with `target: Event<string>` and so on.

Also, check the case, when `source: Event<number>, target: Store<number | string>`